core/kustomize/overlay/openig-nightly/configmap.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: core-deployment-config
+data:
+  BASE_FQDN: "openig-nightly-core.forgerock.financial"
```

Comment:
nightly? or aic?

Reply:
Lol, nope... this is the new name that Shaun had questions over.

It's the IG latest - bleeding-edge build.

Reply:
I had to double check them myself, In the PR it makes even less sense with the removal of nightly-ob|core, but yeah think its alright